### ISSUE TITLE
How to add a new feature to smiles in ConvMolFeaturizer

### ISSUE THREAD

**Issue by sym823808458** [deepchem/deepchem#1538]
Hello everyone,
I am new here in deepchem.I found it a great tool to vectorise the molecules.By using it in my own situation,I found a serious problem that I can only set the column of smiles as the only one feature.For example, I wanna add the quality or mol as a new feature and put them into the deeplearning system.
I know I have to use the ConvMolFeaturizer class,but all I got was an object which means I can't add new float or int into it.
Here is my attempt

```
import deepchem as dc
from rdkit import Chem
smile = 'O=C(C1=CC=C(C=C1)C(O)=O)O'
molecules = []
molecules.append(Chem.MolFromSmiles(smile))

featurizer = dc.feat.graph_features.ConvMolFeaturizer()
mol_object = featurizer.featurize(mols=molecules)
print(mol_object)
```
And I am confused about the shape of the dataset when .here is my code:
```
from rdkit import Chem
from rdkit.Chem.Draw import IPythonConsole
from IPython.display import SVG
from rdkit.Chem import rdDepictor
from rdkit.Chem.Draw import rdMolDraw2D


import numpy as np
import deepchem as dc
from deepchem.molnet import load_tox21

# Only for debug!
np.random.seed(123)

# Load Tox21 dataset
n_features = 1024
tox21_tasks, tox21_datasets, transformers = load_tox21()
train_dataset, valid_dataset, test_dataset = tox21_datasets

train_dataset.get_shape()
```
What I got was ((6264, 1024), (6264, 12), (6264, 12), (6264,))
I do know the 6264 means the number of samples,but what does '12'means?maybe they are X,y,and what else?

**Comment by peastman** [comment]
12 is the number of tasks.  For each of the 6264 molecules, it records the result of 12 different assays.  `get_shape()` returns the shapes of the `X` (features), `y` (labels), `w` (weights), and `ids` arrays, in that order.

**Comment by sym823808458** [comment]
Thanks a lot.It's really helpful.
Could you give me any advice for the first question?I wanna see more infomations about the 'mol_object' in my code.What should I do?Then how can I add new features into it?If it 's impossible or  if you could give me other ways to do the same thing,it will be really nice of you.
Thanks again.

**Comment by VIGS25** [comment]
You can add additional atom properties, by specifying a list of desired properties while initializing the `ConvMolFeaturizer`. 

So this would be something like:
`featurizer = ConvMolFeaturizer(atom_properties=[prop_name])`

**Comment by Jiang-Lu-CAS** [comment]
> Thanks a lot.It's really helpful.
> Could you give me any advice for the first question?I wanna see more infomations about the 'mol_object' in my code.What should I do?Then how can I add new features into it?If it 's impossible or if you could give me other ways to do the same thing,it will be really nice of you.
> Thanks again.

Hi, have you added additional features in ConvMolFeaturizer? How do you find the prop_name? Thank you.
